addons/survey/views/survey_templates.xml

```diff
@@ -97,7 +97,10 @@
                     if question else page.background_image_url
                     if page else survey.background_image_url
                     if survey.background_image_url else False"/>
+        <!-- "action" is set to void in order to avoid it refreshing the page on submit click,
+            before the handler is attached to the button by the JS (can happen on slow connections). -->
         <form role="form" method="post" t-att-name="survey.id"
+                action="javascript:void(0);"
```

Comment:
is it needed ? (the button is disabled, so it should be impossible to trigger the submit event ?)

Reply:
At first I did not think Luc would agree since it causes a slight flicker so I mixed both solutions.

But you are absolutely right, it's not necessary, disabling the buttons is enough :)

Thanks for the review!